src/Uno.UI/KeyboardRectProvider.Android.cs

```diff
@@ -68,20 +71,35 @@ public void Stop()
 		/// </summary>
 		void ViewTreeObserver.IOnGlobalLayoutListener.OnGlobalLayout()
 		{
-			var realMetrics = Get<DisplayMetrics>(_activity.WindowManager.DefaultDisplay.GetRealMetrics);
-			var displayRect = Get<Rect>(_activity.WindowManager.DefaultDisplay.GetRectSize);
-			var usableRect = Get<Rect>(_popupView.GetWindowVisibleDisplayFrame);
+			MeasureRect();
+		}
+
+		public void OnSystemUiVisibilityChange([GeneratedEnum] StatusBarVisibility visibility)
```

Comment:
what's the [GeneratedEnum] for?

Reply:
It's part of the interface
https://developer.xamarin.com/api/member/Android.Views.View+IOnSystemUiVisibilityChangeListener.OnSystemUiVisibilityChange/p/Android.Views.StatusBarVisibility/